otoutfilter/OTOutFilter.go

```diff
@@ -35,13 +36,14 @@ import (
 )
 
 var (
-	logPattern = regexp.MustCompile(`\[(NONE|CRIT|WARN|NOTE|INFO|DEBG)].*\n`)
+	logPattern = regexp.MustCompile(`\d\d:\d\d:\d\d\.\d\d\d \[(-|C|W|N|I|D)].*\n`)
```

Comment:
Are we sure that the OT will compile with `OPENTHREAD_CONFIG_LOG_PREPEND_UPTIME=1`?

Reply:
Probably not, timestamp can be included as optional match.

Reply:
Latest commit simplifies this by not checking timestamp details. Any line detected as a "log line" is now sent to the printLog() function and filtered out as regular node output.